Doc/library/functions.rst

```diff
@@ -1003,12 +1003,11 @@ are always available.  They are listed here in alphabetical order.
    ``'a'``   open for writing, appending to the end of the file if it exists
    ``'b'``   binary mode
    ``'t'``   text mode (default)
-   ``'+'``   open a disk file for updating (reading and writing)
+   ``'+'``   open for updating (reading and writing)
    ========= ===============================================================
 
    The default mode is ``'r'`` (open for reading text, synonym of ``'rt'``).
-   For binary read-write access, the mode ``'w+b'`` opens and truncates the file
-   to 0 bytes.  ``'r+b'`` opens the file without truncation.
+   Mode ``'w+'`` truncates the file.  Mode ``'r+'`` does no truncation.
```

Comment:
What about:
```
   Modes``'w+'`` and ``'w+b'`` open open and truncate the file.  Modes ``'r+'`` and ``'r+b'`` open the file without truncation.
```
?

Reply:
I like it :) Used a slightly different wording.